runtime/entitlements_test.go

```diff
@@ -1107,3 +1107,169 @@ func TestRuntimeCapabilityEntitlements(t *testing.T) {
 	    `)
 	})
 }
+
+func TestRuntimeImportedEntitlementMapInclude(t *testing.T) {
+	t.Parallel()
+
+	storage := newTestLedger(nil, nil)
+	rt := newTestInterpreterRuntime()
+	accountCodes := map[Location][]byte{}
+
+	furtherUpstreamDeployTx := DeploymentTransaction("FurtherUpstream", []byte(`
+        access(all) contract FurtherUpstream {
+            access(all) entitlement X
+			access(all) entitlement Y
+			access(all) entitlement Z
+
+			access(all) entitlement mapping M {
+				X -> Y 
+				Y -> Z
+			}
+        }
```

Comment:
just a nit: string literals here and below seem to have a mix of tabs and whitespaces. Would be nice to have only one of them.